package.json

```diff
@@ -9,14 +9,15 @@
     "README"
   ],
   "scripts": {
-    "lint": "eslint components/** tests/**",
-    "lintfix": "eslint --fix components/** tests/**",
+    "coverage": "codecov -f ./coverage/lcov.info",
+    "lint": "eslint components/ stories/ __tests__/",
+    "lintfix": "npm run lint -- --fix",
     "prepublish": "npm run lint && npm run build",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "storybook": "start-storybook -p 9002",
-    "test": "mocha --require tests/config/setup 'tests/**/*.test.js'",
-    "test:watch": "mocha --require tests/config/setup 'tests/**/*.test.js' --watch",
-    "test:cover": "istanbul cover -x *.test.js _mocha -- -R spec --require tests/config/setup 'tests/**/*.test.js'",
+    "test:story": "jest",
+    "test": "jest && tsc -p typings",
```

Comment:
what does `tsc -p typings` do in our case, since `typings` is already included in the npm release

Reply:
I evaluate the type definition file and check whether it can be compiled with TypeScript.

It is a method recommended mainly by `definitlyTyped`, and this time it also adopts` tsconfig`.

refs: https://github.com/DefinitelyTyped/DefinitelyTyped#how-can-i-contribute